water/lib/Dialect/Wave/IR/WaveAttrs.cpp

```diff
@@ -165,7 +165,7 @@ bool WaveHyperparameterAttr::hasSymbol(StringRef symbolName) const {
 }
 
 //===----------------------------------------------------------------------===//
-// WaveDistributedShapeAttr
+// DistributedShapeAttr
```

Comment:
The rest of the code references this as WaveDistributedShapeAttr so why this change?